DependencyInjection/Configuration.php

```diff
@@ -24,13 +24,16 @@ public function getConfigTreeBuilder(): TreeBuilder
             ->children()
                 ->scalarNode('purger')
                     ->info('ID of the service implementing the \'SofaScore\Purgatory\Purger\PurgerInterface\' interface.')
-                    ->isRequired()
```

Comment:
@zlatkoverk Leave the `isRequired` and `cannotBeEmpty` calls regardless of the `defaultValue` call addition. Those two calls ensure that the processed configuration **needs** to have a value for the specified node.